### PR TITLE
[docs] [reference] update yb_admin and yb_ts_cli reference pages

### LINKED ISSUE #12444: [docs] Inconsistency in yb-admin/yb-ts-cli documentation
### Description

Documentation for [yb-admin](https://docs.yugabyte.com/latest/admin/yb-admin/) and [yb-ts-cli](https://docs.yugabyte.com/latest/admin/yb-ts-cli) currently uses a mix of the following:
```sh
$ yb-admin ...
```
```sh
$ bin/yb-admin ...
```
```sh
yb-admin ...
```
```sh
bin/yb-admin ...
```
This should be made more consistent.

### PR DESCRIPTION
**Pull request by aishwarya24**
Resolves #12444 
Made changes to preview and stable.
**Syntax** should start with `yb_admin` and `yb_ts_cli`.
**Examples** should start with `bin/yb_admin` and `bin/yb_ts_cli`.

@netlify /preview/admin/yb-admin/

### CODE REVIEW

File: docs/content/preview/admin/yb-admin.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ yb-admin \
 To display the online help, run `yb-admin --help` from the YugabyteDB home directory.
 
 ```sh
-$ ./bin/yb-admin --help
+bin/yb-admin --help
```
Comment (lizayugabyte) [inline docs/content/preview/admin/yb-admin.md:44]: I thought the user would need to copy and paste `./bin/yb-admin --help`

File: docs/content/preview/admin/yb-admin.md [hunk 2]
```diff
@@ -166,7 +166,7 @@ yb-admin \
 **Example**
 
 ```sh
-$ ./bin/yb-admin \
+bin/yb-admin \
```
Comment (lizayugabyte) [inline docs/content/preview/admin/yb-admin.md:169]: same comment

File: docs/content/preview/admin/yb-admin.md [hunk 3]
```diff
@@ -208,7 +208,7 @@ yb-admin \
...
```
Comment (lizayugabyte) [inline docs/content/preview/admin/yb-admin.md:211]: same comment

**Review by lizayugabyte — COMMENTED** [review]
Made a couple comments, but they apply everywhere

File: docs/content/preview/admin/yb-admin.md (hunk 1 above)
Comment (aishwarya24) [reply docs/content/preview/admin/yb-admin.md:44]: Yes, Thanks for the clarification again @lizayugabyte . Will make the changes.

**Pull request merged**